Activities/Module13 Activity.md

```diff
@@ -146,3 +146,217 @@ In case your server is is not responding to requests after a long period of inac
 14. Click on the "Environment" tab.
 15. Add a new environment variable called `CLIENT_URL`. For the value, add the client URL (make sure you are adding this env. variable in the server's settings, not the client's). You should now have two environment variables for your server: `MONGDB_URI` and `CLIENT_URL`.
 16. Visit the client URL in your browser to run the application.
+
+
+## Alternative Steps to Set Up Heroku Deployment for Your Backend Service
+
+## Step 1: Enroll in GitHub Student Developer Pack and Heroku for Students
+
+1. **Enroll in GitHub Student Developer Pack**
+   * Visit GitHub Education and select **"Get student benefits"**. Follow the instructions to apply. Approval can take up to 2 business days.
+
+2. **Set Up Heroku for Students**
+   * Go to Heroku.com and create an account (use the same email as your GitHub account).
+   * Enroll in the Heroku for Students offer, which requires the GitHub Student Developer Pack.
+      * **Note**: You'll need to add a credit card for the platform credits that Heroku provides for student use.
+   * After approval, you should see these credits in the billing section of your Heroku account within a few hours. **Wait to create an app until credits are available.**
+
+## Step 2: Create a New Application on Heroku
+
+1. **Navigate to the Heroku Dashboard**
+   * After logging into Heroku, go to your Dashboard.
+
+2. **Create a New App**
+   * Select **New -> Create New App**.
+   * Choose a descriptive app name (this will form part of your app's URL).
+   * Click **Create App** to proceed.
+
+   ![Create New App in Heroku](Module13Assets/CreateNewApp.png)
+
+## Step 3: Deploy the Server from a New Repository
+
+1. **Create a New Repository**
+   * You must create a new Repository on Github which will contain only your server code.
+   * Crate a new Repository on Github and copy paste your server folder.
```

Comment:
Create*

Also, specify "private repository on GitHub" - we don't want the students pushing the code to a public repository until the end of the semester.